mods/d2k/rules/defaults.yaml

```diff
@@ -474,5 +474,6 @@
 	WithColoredOverlay@IDISABLE:
 		RequiresCondition: disabled
 		Palette: disabled
-	GrantConditionOnDisabled@IDISABLE:
+	GrantCondition@IDISABLE:
```

Comment:
Thinking out loud: Might be clearer to have a pair of templates for `^DisableOnLowPower` and `^DisableOnLowPowerOrPowerdown` instead of overriding `GrantCondition@IDISABLE` on powerdown-able actors?  These could then also include `GrantConditionOnPowerState`, `CanPowerDown`, and `WithDecoration@POWERDOWN`.